client/components/deposit-status-chip/style.scss

```diff
@@ -0,0 +1,31 @@
+.deposit-status-chip {
+	@include font-size( 12 );
+	display: inline-block;
+	padding: 0.25rem 0.75rem;
+	border-radius: 1rem;
+	white-space: nowrap;
+
+	&.chip-estimated {
+		background: #f0f0f1;
+		color: #2c3338;
+	}
+
+	&.chip-processing {
+		background: #fcf9e8;
+		color: #614200;
+	}
+
+	&.chip-sent {
+		background: #edfaef;
+		color: #005c12;
+	}
+
+	&.chip-failed {
```

Comment:
Where is the color code for failed status in Figma? I can't find it but it looks OK to me. It looks brownish red.

Reply:
Yeah I couldn't see any red colours in the main Figma designs so I went to the i1 design page and found the red status colour used there.